lib/slax/tentacat/prs.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Slax.Tentacat.Prs do
+  def find(client, org, repo, issue) do
```

Comment:
I only did this because I saw PR's were being handled differently than Issues. Issues were using a behavior, while PR's were directly calling Tentacat.Pulls. Updated PR's to use a behavior.